src/api/component.js

```diff
@@ -179,6 +179,7 @@ export default class extends HTMLElement {
     // DEPRECATED
     //
     // static render()
+    // Note that renderCallback is an optional method!
```

Comment:
Sometimes you don't need to render in leaf components. This just makes it usable for any type of component.